src/elements/pages/analytics/rvt-game-analytics.ts

```diff
@@ -172,7 +171,11 @@ export class RvtGameAnalytics extends LitElement {
 					</div>
 					<div>${moment(start).format('Do MMMM')} - ${moment(end).format('Do MMMM')}</div>
 				</div>
-				${map(this.charts, ({ chart }) => html`<div class="my-4">${chart}</div>`)}
+				${repeat(
```

Comment:
This is just a performance improvement with lit-element, if you ever have a set of items that have a unique key associated with them its better to use `repeat` instead of `map`.